### ISSUE TITLE
Linux: Media keys don't work natively in KDE Plasma 5.15

### ISSUE THREAD

**Issue by thegreyshadow** [jonaski/strawberry#88]
Using: Slackware Linux, 64-bit -current.
Strawberry 0.5.2-53-g384209b compiled from source via a SlackBuild under Plasma 5.15.0, Plasma Frameworks 5.55.0.

Issue:
While volume does work, media keys don't work natively in Plasma desktop. This is strange, since they work very well in Clementine. To have media keys work in Strawberry, I have to enable the setting "Use X11's Shortcut Keys". And I get that scary warning about why using X11 shortcuts is not recommended and all that...

Moreover, while enabling X11's shortcuts, I cannot have KDE Connect's functionality active. That is, playback is not paused when I receive a phone call with my paired device. (I can have this functionality in Clementine, by the way).

Thus, I request if you could please look into this. I really like Strawberry. I don't have any fancy sound hardware but it sounds much better than anything else. Thanks!

**Comment by jonaski** [comment]
I could not reproduce this problem. But X11 shortcuts should be off.
Does mpris work?
Type when playing a song:
qdbus org.mpris.MediaPlayer2.strawberry /org/mpris/MediaPlayer2 org.freedesktop.DBus.Properties.Get org.mpris.MediaPlayer2.Player Metadata

Are you sure you have closed other media players that could grab the same keys? If you used Clementine to make sure it is closed.
Did you try close and re-open? Did you try reboot?

**Comment by alekksander** [comment]
X11 disabled – works like a charm under same system version

**Comment by thegreyshadow** [comment]
Thanks for the pointers.

After the fourth or fifth running/reboot of strawberry, media keys finally worked.
It's not that I did the reboots in rapid succession; it was after four office day sessions (shutdown after the end of the day, boot at the beginning of the next day).

However, before that, nothing that I did worked. And about one of @jonaski questions, I had no other player running whatsoever, and even more, the Media Player Control plasmoid was set to "strawberry".

So, it's strange, took a few tries, but it works now. I will close the report. Thanks again!
